sys/auto_init/auto_init.c

```diff
@@ -29,7 +29,7 @@
 /**
  * @brief   Initializes network devices
  */
-static void _auto_init_gnrc_netif(void)
+void gnrc_netif_auto_init(void)
```

Comment:
I either would move this to their own C-file in `netif/`or to their own sub-module in the `gnrc_netif`. There is no reason to keep it here, now that they are defined by their own module.

Reply:
Ok, moved them to their respective subfolders.